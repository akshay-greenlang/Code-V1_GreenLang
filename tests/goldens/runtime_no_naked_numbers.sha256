a40d5d31fdd79f1842d640bc0c5296f64d841e14e5d4e2032314a472dfae57ee