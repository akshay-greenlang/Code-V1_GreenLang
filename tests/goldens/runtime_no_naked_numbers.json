{
  "message": "Total emissions are 500.00 kgCO2e.",
  "provenance": [
    {
      "path": "$.emissions",
      "quantity": {
        "unit": "kgCO2e",
        "value": 500.0
      },
      "source_call_id": "tc_1"
    }
  ]
}